src/main/java/com/google/sps/data/EntityConstants.java

```diff
@@ -1,23 +1,30 @@
+package com.google.sps.data;
+
 import java.util.Date;
 
-public class EntityConstants {
-  static class AttendeeEntity {
-    public static String TABLE_NAME = "Attendee";
-    public static String SESSION_ID = "SessionId";
-    public static String SCREEN_NAME = "ScreenName";
-    public static String TIME_LAST_POLLED = "TimeLastPolled";
+/** Class that creates constants for tables. **/
+public final class EntityConstants {
+
+  /** Class that creates entity constants for the Attendee table. **/
+  final static class AttendeeEntity {
```

Comment:
These inner classes should be public. I cant remember if that's default for Java